core/src/test/scala/blobstore/AbstractStoreTest.scala

```diff
@@ -177,7 +182,7 @@ trait AbstractStoreTest extends FlatSpec with MustMatchers with BeforeAndAfterAl
       c1 must be(c2)
     }
 
-    test.unsafeRunSync()
+    test
```

Comment:
Double check this is running - `FlatSpec` tests want you to return `Any`, so you won't be warned of a discarded `IO`. Based on my reading I think this test is not being executed